src/NuGet.Clients/VisualStudioAPI/IPackageManagerProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Interface for integrating other package manager.eg:bower,npm
```

Comment:
Interface allowing integration of alternate package manager suggestion for a NuGet package. For example jQuery may appear on Bower and it might be more appropriate to install a package from Bower for certain projects.